projects/igniteui-angular/schematics/ng-add/index.spec.ts

```diff
@@ -103,49 +103,50 @@ describe('ng-add schematics', () => {
     expect(pkgJsonData.dependencies['hammerjs']).toBeTruthy();
   });
 
-  it('should add hammer.js to the main.ts file', async () => {
+  it('should NOT (no longer) add hammer.js to the main.ts file', async () => {
     await runner.runSchematicAsync('ng-add', { normalizeCss: false }, tree).toPromise();
     const mainTs = tree.read(`${sourceRoot}/main.ts`).toString();
-    expect(mainTs).toContain('import \'hammerjs\';');
+    expect(mainTs).not.toContain('import \'hammerjs\';');
   });
 
-  it('should not add hammer.js if it exists in angular.json build options', async () => {
-    const workspace = getWorkspace(tree) as any;
-    const currentProjectName = workspace.defaultProject;
-    workspace.projects[currentProjectName].architect.build.options.scripts.push('./node_modules/hammerjs/hammer.min.js');
-    tree.overwrite('angular.json', JSON.stringify(workspace));
+  it('should NOT (no longer) add hammer.js to the test.ts file', async () => {
     await runner.runSchematicAsync('ng-add', { normalizeCss: false }, tree).toPromise();
+    const testTs = tree.read(`${sourceRoot}/test.ts`).toString();
+    expect(testTs).not.toContain('import \'hammerjs\';');
+  });
 
-    const newContent = tree.read(`${sourceRoot}/main.ts`).toString();
-    expect(newContent.split('import \'hammerjs\';\n// test comment').length).toEqual(1);
+  it('should add hammer.js in angular.json build options under scripts', async () => {
+    await runner.runSchematicAsync('ng-add', { normalizeCss: false }, tree).toPromise();
+    const ngJsonConfigResult = JSON.parse(tree.read('/angular.json').toString());
+    expect(ngJsonConfigResult.projects.testProj.architect.build.options.scripts).toContain('./node_modules/hammerjs/hammer.min.js');
   });
 
-  it('should add hammer.js to the test.ts file', async () => {
+  it('should add hammer.js in angular.json test options under scripts', async () => {
     await runner.runSchematicAsync('ng-add', { normalizeCss: false }, tree).toPromise();
-    const testTs = tree.read(`${sourceRoot}/test.ts`).toString();
-    expect(testTs).toContain('import \'hammerjs\';');
+    const ngJsonConfigResult = JSON.parse(tree.read('/angular.json').toString());
+    expect(ngJsonConfigResult.projects.testProj.architect.test.options.scripts).toContain('./node_modules/hammerjs/hammer.min.js');
   });
 
-  it('should not add hammer.js if it exists in angular.json test options', async () => {
-    const workspace = getWorkspace(tree) as any;
-    const currentProjectName = workspace.defaultProject;
-    workspace.projects[currentProjectName].architect.test.options.scripts.push('./node_modules/hammerjs/hammer.min.js');
-    tree.overwrite('angular.json', JSON.stringify(workspace));
+  it('should NOT add hammer.js in angular.json if it exists in angular.json build options', async () => {
```

Comment:
Something about this duplication :)
```suggestion
  it('should NOT duplicate hammer.js if it exists in angular.json build options', async () => {
```
Also this is missing the point of the original test a bit - it's about not duplicating if hammer's been imported elsewhere too. So if the old logic added to main/test.ts and this test ensured that won't happen if they are already in the workspace angular.json - swapping that around means there should also be a test that checks the new logic won't add to the workspace if the main/test ts have imports already. As far as I see that check remains in the schematic, just need a test for it.